CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Azure Kubernetes Service Changelog
 
+
+## Release 2024-04-11
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+- [Dual-stack networking (IPv4/IPv6) with CNI Overlay](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay?tabs=kubectl#dual-stack-networking) is now generally available (GA) for Linux node pools and does not need the Azure Feature Flag registered on the subscription.
+
+### Release Notes
+
+* Features:
+
+
+* Preview Features:
+
+
+
+
+* Behavioral Changes:
+-   Clusters running Kubernetes 1.29 or later will have kubernetes.azure.com/managedby=aks  label to tigera-operator deployment in Calico clusters 
+- This introduces the constraint template validation behavior change called out in November's release notes
+[2023-11-28](https://github.com/Azure/AKS/releases/tag/2023-11-28) . It also improves cleanup of the addon, as called out in  Issue [#3541](https://github.com/Azure/AKS/issues/3541) , and patches a CVE in the addon.
+- Adds [App routing](https://learn.microsoft.com/en-us/azure/aks/app-routing?tabs=default%2Cdeploy-app-default) to excluded namespaces for policy when the addon is enabled
+- Toggle OSI clusters with 3.1.17 for autoscale
+- Added permissions to nodes/proxy Kubernetes resource for Microsoft.Security/pricings/microsoft-defender-operator role
+- The service mesh add-on now skips validation of its compatibility with cluster version unless mesh upgrade or cluster upgrade is attempted.
+- Using [Alias Minor Version](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#alias-minor-version) is now allowed on preview versions. 
+
+
+* Bug Fixes:
+- Fixes a bug where put agentpool without a specified version in lts clusters would have an internal error.
+- Skips verification of the CNI files installed by the Azure CNS daemonset. This change is made to avoid a race condition observed where components like LinkerD and Istio doing CNI chaining mutate the CNI conflict after its written, causing the validation that the file was written correctly to fail.
+- Error message improved  to specify that it is only allowed to update public SSH key in preview API versions.
+
+* Component Updates: 
+- Upgraded Azure file CSI driver to [v1.29.4](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.29.4)  on AKS 1.28, [v1.30.1](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.30.1)  on AKS 1.29
+- Upgraded Azure Blob CSI driver to [v1.23.4](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.23.4)  on AKS 1.28 and 1.29
+- Upgraded extension manager to [v1.15.3](https://github.com/azure-core/ClusterConfigurationAgent/releases/tag/v1.15.3)
```

Comment:
we need to remove this line. will add a comment on our internal chat for more details